clusterman/draining/kubernetes.py

```diff
@@ -33,3 +33,17 @@ def uncordon(connector: Optional[KubernetesClusterConnector], agent_id: str) ->
     else:
         log.info(f"Unable to uncordon {agent_id} (no Kubernetes connector configured)")
         return False
+
+
+def clean_node(connector: Optional[KubernetesClusterConnector], agent_id: str) -> bool:
+    """Cordons and forcibly delete all tasks from a given node.
```

Comment:
Just to clarify, this means it doesn't care about PDBs, right?

Reply:
yeap, definitely! We use clean_node when we received spot termination warning form AWS. We should remove all pods asap. Because AWS terminates nodes in 2 minutes.